tree/30_generic_methods/sharedfile_from_node.cf

```diff
@@ -0,0 +1,48 @@
+#####################################################################################
+# Copyright 2013 Normation SAS
+#####################################################################################
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, Version 3.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+#####################################################################################
+
+# @name Sharedfile from node
+# @description This method retreive a file shared from another rudder node
+# @documentation This method eanotheve a file shared fromr rudder node using a unique file identifier.
+#  The file will be downloaded using cfengine protocol and copied into a new file.
```

Comment:
Could you also add an example (to see what id look like, that the path is absolute, etc.)